Provides support to decode flags of the %ccr register.
Submitted to upstream as:
https://sourceware.org/bugzilla/show_bug.cgi?id=20928

--- gdb-7.11/gdb/sparc-tdep.h.orig	2016-11-13 21:53:36.828489462 -0800
+++ gdb-7.11/gdb/sparc-tdep.h	2016-11-13 21:45:25.465272567 -0800
@@ -76,6 +76,7 @@
   /* ISA-specific data types.  */
   struct type *sparc_psr_type;
   struct type *sparc_fsr_type;
+  struct type *sparc64_ccr_type;
   struct type *sparc64_pstate_type;
   struct type *sparc64_fsr_type;
   struct type *sparc64_fprs_type;
